utils/installer/config.example.lua

```diff
@@ -18,7 +18,7 @@ lvim.leader = "space"
 -- add your own keymapping
 lvim.keys.normal_mode["<C-s>"] = ":w<cr>"
 -- unmap a default keymapping
--- lvim.keys.normal_mode["<C-Up>"] = ""
+-- lvim.keys.normal_mode["<C-Up>"] = false
```

Comment:
```suggestion
-- vim.cmd [[ unmap <C-Up> ]]
```

Reply:
Setting to false also functions properly still, so I guess do we want to advocate a Lua-based or a Vimscript-based solution like the above?

Reply:
Since it's an example config, I'd say there could be a case for including both so that the user can pick the one that works best for them.  But, there's also a case for picking the most lvim-best-practices way.